primordialsoup/static/css/primordialsoup.css

```diff
@@ -65,6 +65,12 @@ body {
     src: url(/static/font/Basteleur-Moonlight.otf);
 }
 
+@font-face {
+    font-family: basteleurbold;
```

Comment:
Tror bare den her skal hedde det samme som den anden `basteleur`, så bruger den automatisk bold versionen her hvis man bruger font-weight bold eller <b>

Reply:
Nej -det er to forskellige (Moonlight og Bold). Jeg bruger den til '+' og '-' i zoom knapperne til kortet. Se: https://velvetyne.fr/fonts/basteleur/

Reply:
<img width="280" alt="Screenshot 2024-05-23 at 14 56 01" src="https://github.com/ursuppeeppusru/ursuppe/assets/43446790/2a24132f-14ec-43ab-98b8-981860e78634">

Reply:
Ah okay, i princippet en helt anden font så?

Reply:
New push :)